packages/global/components/site-header/contextual.marko

```diff
@@ -4,11 +4,10 @@ import { asArray } from "@parameter1/base-cms-utils";
 
 $ const { config, site, req } = out.global;
 
-$ const blockName = input.blockName || "site-header";
+$ const { blockName = "site-header", pageType } = input;
 
 $ const contexts = site.getAsArray("navigation.contexts");
-$ const contextualNav = contexts.find(({ when }) => asArray(when).some(path => isActiveLink(req.path, path)));
-
+$ const contextualNav = (pageType === "website-section") ?  contexts.find(({ when }) => asArray(when).some(path => isActiveLink(req.path, path))) : null;
```

Comment:
Check the whitespacing near the ternary (`?  w`), but I'd recommend breaking this line apart for better legibility. Something like this...

```js
$ const useContext = pageType === "website-section";
$ const isActive = (conditions) => conditions.some((path) => isActiveLink(req.path, path));
$ const contextualNav = useContext && contexts.find((c) => isActive(getAsArray(c, "when")));
```